catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickTestUtils.java

```diff
@@ -105,4 +106,16 @@ public static void createUserListFromDataFragment(String userListName, boolean f
 		onView(withText(userListName))
 				.check(matches(isDisplayed()));
 	}
+
+	public static void deleteBrickAtPosition(int position, int intBrickResouce) {
+		onScriptList().atPosition(position).onChildView(withText(intBrickResouce)).perform(click());
+		onView(withText(R.string.brick_context_dialog_delete_brick))
+				.perform(click());
+		onView(withText(R.string.yes))
+				.perform(click());
+	}
+
+	public static void checkBrickNotExists(int intResourceId) {
```

Comment:
remove this.

no functions with only 1 line of code.
this is obfuscating the code, it makes it only harder to read.